### PR TITLE
 [PT-BR] Update `Game Mode`

### PR DESCRIPTION
**Pull request by slowayy**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by ItaloSaid** [comment]
LGTM!

File: wiki/Game_mode/pt-br.md [hunk 1]
```diff
@@ -3,28 +3,57 @@ outdated_translation: true
 outdated_since: d5b45241bf64c7abeadfa160238f428bbcaa4d2a
 ---
```
Comment (Sitoria) [inline wiki/Game_mode/pt-br.md:4]: if this translation is translated on latest en.md, then this must be deleted
erase line 1 to 5

File: wiki/Game_mode/pt-br.md
Comment (Sitoria) [inline wiki/Game_mode/pt-br.md:1]: Title must be [PT-BR] Update `Game mode`
use ` <- this one in front of Game and end of mode

**Review by Sitoria — CHANGES_REQUESTED** [review]
erase outdate and title issue

File: wiki/Game_mode/pt-br.md (hunk 1 above)
Comment (slowayy) [reply wiki/Game_mode/pt-br.md]: ok!

**Review by Sitoria — APPROVED** [review]
good to go!

**Comment by Walavouchey** [comment]
force-pushed to remove "merge master" spam

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**